<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>373</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>7</number>
    </property>
    <property name="topMargin">
     <number>7</number>
    </property>
    <property name="rightMargin">
     <number>7</number>
    </property>
    <property name="bottomMargin">
     <number>7</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="GfxEditorView" name="view">
         <property name="minimumSize">
          <size>
           <width>624</width>
           <height>288</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="interactive">
          <bool>true</bool>
         </property>
         <property name="dragMode">
          <enum>QGraphicsView::NoDrag</enum>
         </property>
         <property name="transformationAnchor">
          <enum>QGraphicsView::AnchorViewCenter</enum>
         </property>
         <property name="resizeAnchor">
          <enum>QGraphicsView::NoAnchor</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="sectorDec">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sectorSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sectorInc">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Map</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>7</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>7</number>
          </property>
          <property name="bottomMargin">
           <number>7</number>
          </property>
          <item alignment="Qt::AlignRight">
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="checkDelete">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteMap">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="newMap">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>Create ...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mapBox"/>
          </item>
         </layout>
         <zorder>mapBox</zorder>
         <zorder>widget_8</zorder>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Editor</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>7</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>7</number>
          </property>
          <property name="bottomMargin">
           <number>7</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QFormLayout" name="formLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkFineGrid">
               <property name="text">
                <string>Fine Grid</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="checkMainGrid">
               <property name="text">
                <string>Main Grid</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="checkWall">
               <property name="text">
                <string>Walls</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="checkPlayer">
               <property name="text">
                <string>Player</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Sector</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>7</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>7</number>
          </property>
          <property name="bottomMargin">
           <number>7</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="sector">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Courier New</family>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Go</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scripts">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Edit Scripts ...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="btnApply">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnReload">
               <property name="text">
                <string>Reload</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSketches">
    <property name="title">
     <string>Sketches</string>
    </property>
    <addaction name="actionSketchesBgTiles"/>
    <addaction name="actionSketchesCnfTiles"/>
    <addaction name="actionSketchesCelinigs"/>
    <addaction name="actionSketchesFloors"/>
    <addaction name="actionSketchesWalls"/>
    <addaction name="separator"/>
    <addaction name="actionSketchesOpenAll"/>
    <addaction name="actionSketchesCloseAll"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEditSceneries"/>
    <addaction name="actionEditThemes"/>
    <addaction name="actionScripts"/>
    <addaction name="actionEditHumans"/>
    <addaction name="actionEditCybers"/>
    <addaction name="actionEditMusics"/>
    <addaction name="actionEditEnemies"/>
   </widget>
   <widget class="QMenu" name="menuScenery">
    <property name="title">
     <string>Scenery</string>
    </property>
    <addaction name="actionSceneryCharSet"/>
    <addaction name="actionSceneryBgTiles"/>
    <addaction name="actionSceneryCnfTiles"/>
    <addaction name="actionScenerySprites"/>
    <addaction name="actionSceneryWalls"/>
    <addaction name="separator"/>
    <addaction name="actionSceneryOpenAll"/>
    <addaction name="actionSceneryCloseAll"/>
   </widget>
   <widget class="QMenu" name="menuMap">
    <property name="title">
     <string>Map</string>
    </property>
    <addaction name="actionMapSettings"/>
    <addaction name="actionMapCyberSpace"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSketches"/>
   <addaction name="menuScenery"/>
   <addaction name="menuMap"/>
  </widget>
  <action name="actionGlobalCeiling">
   <property name="text">
    <string>Ceiling Sprite</string>
   </property>
  </action>
  <action name="actionGlobalFloor">
   <property name="text">
    <string>Floor Sprite</string>
   </property>
  </action>
  <action name="actionGlobalWall">
   <property name="text">
    <string>Wall</string>
   </property>
  </action>
  <action name="actionGlobalSketch">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tile Sketch</string>
   </property>
  </action>
  <action name="actionGlobalCnFSketch">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CnF Tile Sketch</string>
   </property>
  </action>
  <action name="actionSceneryBlock">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Block</string>
   </property>
  </action>
  <action name="actionSceneryTile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionSceneryAnimatedTile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Animated Tile</string>
   </property>
  </action>
  <action name="actionSceneryCnFTile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CnF Tile</string>
   </property>
  </action>
  <action name="actionScenerySprite">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sprite</string>
   </property>
  </action>
  <action name="actionSceneryWall">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Wall</string>
   </property>
  </action>
  <action name="actionAllBrowsersOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open all</string>
   </property>
  </action>
  <action name="actionAllBrowsersClose">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionGlobalOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open all</string>
   </property>
  </action>
  <action name="actionGlobalClose">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionSceneryOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open all</string>
   </property>
  </action>
  <action name="actionSceneryClose">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionGlobalPlayer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Human</string>
   </property>
  </action>
  <action name="actionGlobalCyber">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cyber</string>
   </property>
  </action>
  <action name="actionGlobalEnemy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enemy</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEditHumans">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Humans</string>
   </property>
  </action>
  <action name="actionEditCybers">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cybers</string>
   </property>
  </action>
  <action name="actionEditEnemies">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enemies</string>
   </property>
  </action>
  <action name="actionEditThemes">
   <property name="text">
    <string>Themes</string>
   </property>
  </action>
  <action name="actionEditSceneries">
   <property name="text">
    <string>Sceneries</string>
   </property>
  </action>
  <action name="actionChar_Set">
   <property name="text">
    <string>Char Set</string>
   </property>
  </action>
  <action name="actionEditMusics">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Musics</string>
   </property>
  </action>
  <action name="actionSceneryCharSet">
   <property name="text">
    <string>Char Set</string>
   </property>
  </action>
  <action name="actionSceneryBgTiles">
   <property name="text">
    <string>Bg Tiles</string>
   </property>
  </action>
  <action name="actionSceneryCnfTiles">
   <property name="text">
    <string>CnF Tiles</string>
   </property>
  </action>
  <action name="actionScenerySprites">
   <property name="text">
    <string>Sprites</string>
   </property>
  </action>
  <action name="actionSceneryWalls">
   <property name="text">
    <string>Walls</string>
   </property>
  </action>
  <action name="actionSceneryOpenAll">
   <property name="text">
    <string>Open all</string>
   </property>
  </action>
  <action name="actionSceneryCloseAll">
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionSketchesOpenAll">
   <property name="text">
    <string>Open all</string>
   </property>
  </action>
  <action name="actionSketchesCloseAll">
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionSketchesBgTiles">
   <property name="text">
    <string>Bg Tile Sketches</string>
   </property>
  </action>
  <action name="actionSketchesCnfTiles">
   <property name="text">
    <string>CnF Tile Sketches</string>
   </property>
  </action>
  <action name="actionSketchesCelinigs">
   <property name="text">
    <string>Celinig Sprites</string>
   </property>
  </action>
  <action name="actionSketchesFloors">
   <property name="text">
    <string>Floor Sprites</string>
   </property>
  </action>
  <action name="actionSketchesWalls">
   <property name="text">
    <string>Walls</string>
   </property>
  </action>
  <action name="actionMapSettings">
   <property name="text">
    <string>Settings ...</string>
   </property>
  </action>
  <action name="actionMapCyberSpace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cyber Space ...</string>
   </property>
  </action>
  <action name="actionMapScripts">
   <property name="text">
    <string>Shared Scripts ...</string>
   </property>
  </action>
  <action name="actionScripts">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scripts</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GfxEditorView</class>
   <extends>QGraphicsView</extends>
   <header>gfx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
